source/sass/mixins/_spacing.scss

```diff
@@ -132,3 +132,24 @@
   list-style: none;
   @include padding(0, inline-start);
 }
+
+@mixin inset-inline($start: null, $end: null) {
+  left: $start; // stylelint-disable-line csstools/use-logical
+  right: $end; // stylelint-disable-line csstools/use-logical
+
+  @supports (inset-inline-start: $start) {
```

Comment:
When no value is passed for `$start`, this will generate the CSS `@supports (inset-inline-start: null) {`. Would be better to test the actual value passed, and check for support of `inset-inline-end` if `$end` value passed but no `$start` value has been passed.